frontend/test/playwright/e2e/load-more.spec.ts

```diff
@@ -125,6 +130,32 @@ test.describe("Load more button", () => {
         })
         await expect(page.locator(loadMoreButton)).not.toBeVisible()
       })
+
+      /**
+       * Checks that an analytics event is posted to /api/event and has the correct
+       * payload for the REACH_RESULT_END event.
+       */
+      test(`Sends a valid analytics event when user reaches the load more page`, async ({
```

Comment:
Should we have a second test or a second piece to this test to ensure that the event is fired off again after load more is clicked and that the event appropriately has page 2 in the payload?

Reply:
I'll look into adding one, I think that would be wise.

Reply:
Done 👍